spec/index.html

```diff
@@ -153,1920 +183,1796 @@ <h2>Semantic Extensions and Entailment Regimes</h2>
+          </div>
+        </div>
+      </section>
+      <section class="notoc" id="RDFS_axiomatic_triples">
+        <h6 class="defcap">RDFS Axiomatic Triples</h6>
+        <div class="ruletable">
+          <p><code>
+            rdf:type rdfs:domain rdfs:Resource .<br>
+            rdfs:domain rdfs:domain rdf:Property .<br>
+            rdfs:range rdfs:domain rdf:Property .<br>
+            rdfs:subPropertyOf rdfs:domain rdf:Property .<br>
+            rdfs:subClassOf rdfs:domain rdfs:Class .<br>
+            rdf:subject rdfs:domain rdf:Statement .<br>
+            rdf:predicate rdfs:domain rdf:Statement .<br>
+            rdf:object rdfs:domain rdf:Statement .<br>
+            rdfs:member rdfs:domain rdfs:Resource .<br>
+            rdf:first rdfs:domain rdf:List .<br>
+            rdf:rest rdfs:domain rdf:List .<br>
+            rdfs:seeAlso rdfs:domain rdfs:Resource .<br>
+            rdfs:isDefinedBy rdfs:domain rdfs:Resource .<br>
+            rdfs:comment rdfs:domain rdfs:Resource .<br>
+            rdfs:label rdfs:domain rdfs:Resource .<br>
+            rdf:value rdfs:domain rdfs:Resource .<br>
+            <br>
+            rdf:type rdfs:range rdfs:Class .<br>
+            rdfs:domain rdfs:range rdfs:Class .<br>
+            rdfs:range rdfs:range rdfs:Class .<br>
+            rdfs:subPropertyOf rdfs:range rdf:Property .<br>
+            rdfs:subClassOf rdfs:range rdfs:Class .<br>
+            rdf:subject rdfs:range rdfs:Resource .<br>
+            rdf:predicate rdfs:range rdfs:Resource .<br>
+            rdf:object rdfs:range rdfs:Resource .<br>
+            rdfs:member rdfs:range rdfs:Resource .<br>
+            rdf:first rdfs:range rdfs:Resource .<br>
+            rdf:rest rdfs:range rdf:List .<br>
+            rdfs:seeAlso rdfs:range rdfs:Resource .<br>
+            rdfs:isDefinedBy rdfs:range rdfs:Resource .<br>
+            rdfs:comment rdfs:range rdfs:Literal .<br>
+            rdfs:label rdfs:range rdfs:Literal .<br>
+            rdf:value rdfs:range rdfs:Resource .<br>
+            <br>
+            rdf:Alt rdfs:subClassOf rdfs:Container .<br>
+            rdf:Bag rdfs:subClassOf rdfs:Container .<br>
+            rdf:Seq rdfs:subClassOf rdfs:Container .<br>
+            rdfs:ContainerMembershipProperty rdfs:subClassOf rdf:Property .<br>
+            <br>
+            rdfs:isDefinedBy rdfs:subPropertyOf rdfs:seeAlso .<br>
+            <br>
+            rdfs:Datatype rdfs:subClassOf rdfs:Class .<br>
+            <br>
+            rdf:_1 rdf:type rdfs:ContainerMembershipProperty .<br>
+            rdf:_1 rdfs:domain rdfs:Resource .<br>
+            rdf:_1 rdfs:range rdfs:Resource .<br>
+            rdf:_2 rdf:type rdfs:ContainerMembershipProperty .<br>
+            rdf:_2 rdfs:domain rdfs:Resource .<br>
+            rdf:_2 rdfs:range rdfs:Resource .<br>
```

Comment:
This should be re-marked as a `<pre>` block, and the columns should be visually aligned, as this significantly increases comprehensibility of such blocks.